scaphandre/README.md

```diff
@@ -0,0 +1,68 @@
+# Agent Check: Scaphandre
```

Comment:
I think we usually just call these by the service name:

```suggestion
# Scaphandre
```

Reply:
Done.